bindings/cs/cs.sln

```diff
@@ -24,42 +24,42 @@ Global
 		HideSolutionNode = FALSE
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{90EA4DBB-69E6-426D-985A-C66D65A2E63A}.Debug|Any CPU.ActiveCfg = Debug|x64
-		{90EA4DBB-69E6-426D-985A-C66D65A2E63A}.Debug|Any CPU.Build.0 = Debug|x64
-		{90EA4DBB-69E6-426D-985A-C66D65A2E63A}.Debug|x64.ActiveCfg = Debug|x64
-		{90EA4DBB-69E6-426D-985A-C66D65A2E63A}.Debug|x64.Build.0 = Debug|x64
-		{90EA4DBB-69E6-426D-985A-C66D65A2E63A}.Debug|x86.ActiveCfg = Debug|x64
-		{90EA4DBB-69E6-426D-985A-C66D65A2E63A}.Debug|x86.Build.0 = Debug|x64
-		{90EA4DBB-69E6-426D-985A-C66D65A2E63A}.Release|Any CPU.ActiveCfg = Release|x64
-		{90EA4DBB-69E6-426D-985A-C66D65A2E63A}.Release|Any CPU.Build.0 = Release|x64
-		{90EA4DBB-69E6-426D-985A-C66D65A2E63A}.Release|x64.ActiveCfg = Release|x64
-		{90EA4DBB-69E6-426D-985A-C66D65A2E63A}.Release|x64.Build.0 = Release|x64
-		{90EA4DBB-69E6-426D-985A-C66D65A2E63A}.Release|x86.ActiveCfg = Release|x64
-		{90EA4DBB-69E6-426D-985A-C66D65A2E63A}.Release|x86.Build.0 = Release|x64
-		{F2D29C5D-DA83-456D-994F-57AB43EA9470}.Debug|Any CPU.ActiveCfg = Debug|x64
-		{F2D29C5D-DA83-456D-994F-57AB43EA9470}.Debug|Any CPU.Build.0 = Debug|x64
-		{F2D29C5D-DA83-456D-994F-57AB43EA9470}.Debug|x64.ActiveCfg = Debug|x64
-		{F2D29C5D-DA83-456D-994F-57AB43EA9470}.Debug|x64.Build.0 = Debug|x64
-		{F2D29C5D-DA83-456D-994F-57AB43EA9470}.Debug|x86.ActiveCfg = Debug|x64
-		{F2D29C5D-DA83-456D-994F-57AB43EA9470}.Debug|x86.Build.0 = Debug|x64
-		{F2D29C5D-DA83-456D-994F-57AB43EA9470}.Release|Any CPU.ActiveCfg = Release|x64
-		{F2D29C5D-DA83-456D-994F-57AB43EA9470}.Release|Any CPU.Build.0 = Release|x64
-		{F2D29C5D-DA83-456D-994F-57AB43EA9470}.Release|x64.ActiveCfg = Release|x64
-		{F2D29C5D-DA83-456D-994F-57AB43EA9470}.Release|x64.Build.0 = Release|x64
-		{F2D29C5D-DA83-456D-994F-57AB43EA9470}.Release|x86.ActiveCfg = Release|x64
-		{F2D29C5D-DA83-456D-994F-57AB43EA9470}.Release|x86.Build.0 = Release|x64
-		{536E0590-7287-4D73-A5D7-5B8EFD2945B8}.Debug|Any CPU.ActiveCfg = Debug|x64
-		{536E0590-7287-4D73-A5D7-5B8EFD2945B8}.Debug|Any CPU.Build.0 = Debug|x64
-		{536E0590-7287-4D73-A5D7-5B8EFD2945B8}.Debug|x64.ActiveCfg = Debug|x64
-		{536E0590-7287-4D73-A5D7-5B8EFD2945B8}.Debug|x64.Build.0 = Debug|x64
-		{536E0590-7287-4D73-A5D7-5B8EFD2945B8}.Debug|x86.ActiveCfg = Debug|x64
-		{536E0590-7287-4D73-A5D7-5B8EFD2945B8}.Debug|x86.Build.0 = Debug|x64
-		{536E0590-7287-4D73-A5D7-5B8EFD2945B8}.Release|Any CPU.ActiveCfg = Release|x64
-		{536E0590-7287-4D73-A5D7-5B8EFD2945B8}.Release|Any CPU.Build.0 = Release|x64
-		{536E0590-7287-4D73-A5D7-5B8EFD2945B8}.Release|x64.ActiveCfg = Release|x64
-		{536E0590-7287-4D73-A5D7-5B8EFD2945B8}.Release|x64.Build.0 = Release|x64
-		{536E0590-7287-4D73-A5D7-5B8EFD2945B8}.Release|x86.ActiveCfg = Release|x64
-		{536E0590-7287-4D73-A5D7-5B8EFD2945B8}.Release|x86.Build.0 = Release|x64
+		{90EA4DBB-69E6-426D-985A-C66D65A2E63A}.Debug|Any CPU.ActiveCfg = Debug|Any CPU

```

Comment:
It looks fine. I had the same change on my VS 2019 instance. But if your team does not use 2019 yet, then definitely revert.

The most important thing is that when debugging/testing that the runtime arch matches the libmongocrypt binary. It is often in previous versions of VS that it makes 32-bit DLLs which cannot load 64-bit C++ dlls as a result and you get an error in the LibraryLoader.

Reply:
We used to build of `x64` only... now you are building for `Any CPU`.

Is that intentional?

Reply:
Yes and no. This change is added automatically by vs. If we just replace a new `Any CPU` value on `x64`, the solution won't be opened in vs2019. 
I was able to refactor these lines and then make the solution to be opened with saving only x64 conditions. I did it by removing some of the lines. But I'm not sure that my manual changes were safe. So, I would leave it as is since it's definitely not related to the scope of the ticket directly and looks like some migration logic from vs itself.

Reply:
I actually don't understand this change at all. I will defer to other reviewers.

Reply:
https://github.com/mongodb/libmongocrypt/pull/110#discussion_r503340405

Reply:
Yes, this is why I explained it in the first comment.